src/kv/2pc.cc

```diff
@@ -1,23 +1,66 @@
 #include <pingcap/kv/2pc.h>
 #include <pingcap/kv/RegionClient.h>
 #include <pingcap/kv/Txn.h>
+#include <pingcap/pd/Oracle.h>
 
 namespace pingcap
 {
 namespace kv
 {
 
-TwoPhaseCommitter::TwoPhaseCommitter(Txn * txn) : log(&Logger::get("pingcap.tikv"))
+constexpr uint64_t managedLockTTL = 20000; // 20s
+
+constexpr uint64_t bytesPerMiB = 1024 * 1024;
+
+//constexpr uint64_t ttlManagerRunThreshold = 32 * 1024 * 1024;
+
+// TODO: use the right ttlManagerRunThreshold
+// use a small threshold to simplify test here
+constexpr uint64_t ttlManagerRunThreshold = 512 * 1024;
```

Comment:
It's not ok to hack like this. we can use  const uint64_t to represent it, and use `const_cast` to change const value in test.

Reply:
Fixed.